api/badgesize.ts

```diff
@@ -7,29 +7,32 @@ export default createBadgenHandler({
     '/badgesize/normal/amio/emoji.json/master/emoji-compact.json': 'normal size',
     '/badgesize/brotli/amio/emoji.json/master/emoji-compact.json': 'brotli size',
     '/badgesize/gzip/amio/emoji.json/master/emoji-compact.json': 'gzip size',
-    '/badgesize/normal/https/unpkg.com/snarkdown/dist/snarkdown.js': 'arbitrary url',
+    '/badgesize/normal/file-url/https/unpkg.com/snarkdown/dist/snarkdown.js': 'arbitrary url',
+    '/badgesize/normal/file-url/unpkg.com/snarkdown/dist/snarkdown.js': 'arbitrary url',
```

Comment:
Hiding _legacy_ invocation and advertising `/file-url` approach together with URL shorthand.